apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-site-editor-document-actions-dropdown-click.js

```diff
@@ -0,0 +1,66 @@
+import tracksRecordEvent from './track-record-event';
+
+/**
+ * Return the event definition object to track `wpcom-site-editor-document-actions-dropdown-clicks`.
+ *
+ * @returns {import('./types').DelegateEventHandler} event object definition.
+ */
+export const wpcomSiteEditorDocumentActionsDropdownOpen = () => ( {
+	id: 'wpcom-site-editor-document-actions-dropdown-open',
+	selector: '.components-dropdown .edit-site-document-actions__get-info',
+	type: 'click',
+	handler: () => tracksRecordEvent( 'wpcom_site_editor_document_actions_dropdown_open' ),
+} );
+
+/**
+ * Return the event definition object to track `wpcom-site-editor-document-actions-template-area-click`.
+ *
+ * @returns {import('./types').DelegateEventHandler} event object definition.
+ */
+export const wpcomSiteEditorDocumentActionsTemplateAreaClick = () => ( {
+	id: 'wpcom-site-editor-document-actions-template-area-click',
+	selector: '.components-dropdown__content .edit-site-template-details__template-areas',
+	type: 'click',
+	handler: ( event ) => {
+		const headerSVGSnippet =
+			'M18.5 10.5H10v8h8a.5.5 0 00.5-.5v-7.5zm-10 0h-3V18a.5.5 0 00.5.5h2.5v-8zM6';
+		const footerSVGSnippet =
+			'M18 5.5h-8v8h8.5V6a.5.5 0 00-.5-.5zm-9.5 8h-3V6a.5.5 0 01.5-.5h2.5v8zM6';
+
+		// There are no selectors that provide a distinction between the Header and Footer template
+		// area buttons. We could use text content instead, but that information is unreliable when
+		// the text is translated. Because of this, we look for accompanying SVG icons and derive
```

Comment:
>  We could use text content instead, but that information is unreliable when the text is translated.

If we use the same translate function/data to check, shouldn't they match up? That is if the items in the dropdown are using `__( 'Header' )`, can we check to see if it is equal to `__( 'Header' )` here and expect them to reliably line up?

Reply:
I'm unsure. I gave it some brief consideration, but in the FG they explicitly state that "Relying on text content to determine data for the Tracks event is unreliable due to i18n. If strings are parsed from the event’s DOM, then the relevant source code should be inspected to determine whether the string is translatable. If it is, then this may affect the accuracy and/or utility of your Tracks event across different locales." ( [source](PCYsg-nrf-p2) ).

I assumed that it wouldn't behave properly

Reply:
I think it **should** work if they we are inspecting the source code and ensuring we are using the same translation maps when comparing, but also I don't think it really buys us anything here. Whether we are relying on SVGs or text doesn't change the fact that these events wont get the right context/area if what we rely on changes. I think the big thing is having a test to catch any regression, and don't think SVG vs. translated text will make much of a difference. - so we can probably just stick with this SVG check.